README.md

```diff
@@ -0,0 +1,65 @@
+
```

Comment:
You can also add how to use binaries instead. I mean no need to download and run in Go. You can head towards releases tab -> download binary based on your OS distro. Mention this in README as well. It's more easier for people without needing to install Go on their machine.